drivers/soundwire/intel.c

```diff
@@ -745,10 +745,10 @@ static int intel_free_stream(struct sdw_intel *sdw,
  * bank switch routines
  */
 
-static int intel_pre_bank_switch(struct sdw_bus *bus)
```

Comment:
@plbossart i  know i may be asking for too much but if you had a separate patch for using each new op, that would make it easier to tell if something was done wrong easily

Reply:
humm, why not. It's true that the next-to-last patch is quite invasive. That would add about 10 patches but they'd be smaller.